p03-form-validations/src/Form/Form.services.js

```diff
@@ -3,31 +3,36 @@ export const fieldsList = [
     name: "name",
     label: "User first name",
     type: "text",
+    value: '',
```

Comment:
it's complete valid to manually add the value prop to the fieldList array, just to let you know that you could added when you set the fieldList. Example: 

```
const [fieldList, setFieldList] = useState(fieldsList.map( field => ({ ...field, value: "" }) )
```